src/clients/node/scripts/publish.sh

```diff
@@ -15,12 +15,12 @@ if [ "${exists}" = "true" ]; then
     echo "Package tigerbeetle-node@${PACKAGE_JSON_VERSION} already exists - incrementing patch version from latest on npm (${NPM_LATEST_VERSION}) to (${NPM_PLUS_ONE_VERSION})"
 
     rm -rf /tmp/change-package-version && mkdir -p /tmp/change-package-version
-    tar -xzpf tigerbeetle-node-*.tgz -C /tmp/change-package-version
+    tar -xzf tigerbeetle-node-*.tgz -C /tmp/change-package-version
 
     jq ".version=\"${NPM_PLUS_ONE_VERSION}\"" /tmp/change-package-version/package/package.json > /tmp/change-package-version/package/package.json-new
     mv -f /tmp/change-package-version/package/package.json-new /tmp/change-package-version/package/package.json
 
-    tar -czpf tigerbeetle-node-updated.tgz -C /tmp/change-package-version/package .
+    tar -czf tigerbeetle-node-updated.tgz -C /tmp/change-package-version package
```

Comment:
Does npm publish have a dry-run command we could set up for PR runs so we can make changes to this in the future without needing to merge to test?

Reply:
It does have `--dry-run` - but in this case, I was doing this package / repackage hack for no good reason (as @matklad pointed out in the original PR :joy:)

Reply:
It would be great to shift to environment secrets, then we can use dry runs, etc and be really confident that even if somehow something goes wrong there, the pipeline not running on main won't have access to the secrets, so it won't be able to publish